lib/auth/plugins.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth
+
+import (
+	"github.com/gravitational/teleport-plugins/access/common/auth/oauth"
```

Comment:
Do you plan to import other things besides that interface?
If the answer is no, we probably can drop the import locally since it's just importing an interface def and we prefer local interfaces

Reply:
`teleport.e` will import more things from `teleport-plugins` (the concrete plugin types themselves)